apps/di/src/app/app.component.ts

```diff
@@ -23,10 +23,31 @@ export class ProductDirective {
   }
 }
 
+@Directive({
+  selector: 'tr[currencyCode]',
+  standalone: true,
+  providers: [{ provide: CurrencyService }],
```

Comment:
question: I have never seen this synthax, what is the difference with 
```ts
providers: [CurrencyService]
```

Reply:
Before I settled on the current solution I was trying some approaches using providers `multi: true` and I forgot to revert this back to `providers: [CurrencyService]`. Fixed! Thanks for reviewing.